types/office-js-preview/index.d.ts

```diff
@@ -11691,7 +11691,9 @@ declare namespace Office {
          */
         getEntitiesByType(entityType: MailboxEnums.EntityType | string): (string | Contact | MeetingSuggestion | PhoneNumber | TaskSuggestion)[];
         /**
-         * Returns well-known entities in the selected item that pass the named filter defined in the manifest XML file.
+         * Returns well-known entities in the selected item that pass the named filter defined in an XML manifest file.
+         * 
+         * **Note**: This method is used with the activation rules feature for Outlook Add-ins, which is not supported by the {@link https://learn.microsoft.com/en-us/office/dev/add-ins/develop/json-manifest-overview | Teams manifest for Office Add-ins (preview)}.
```

Comment:
Here and other instances
```suggestion
         * **Note**: This method is used with the {@link https://learn.microsoft.com/office/dev/add-ins/outlook/activation-rules | activation rules feature for Outlook add-ins}, which isn't supported by the {@link https://learn.microsoft.com/office/dev/add-ins/develop/json-manifest-overview | Teams manifest for Office Add-ins (preview)}.
```